dashboard/app/views/levels/editors/fields/_validation_code.html.haml

```diff
@@ -18,7 +19,13 @@
   -# GamelabJr, Dancelab, and Poetry are also Gamelab levels
   - if @level.is_a?(Gamelab)
     %p
-      This is a snippet of javascript that is run after every draw loop. Call levelSuccess() in here to stop the level as a success. You can also pass a number into levelSuccess() to give the student a non-perfect test result (see <a href="https://github.com/code-dot-org/code-dot-org/blob/101fbfd2dcdd635d7359e991559ba782f9fd00be/apps/src/constants.js#L33-L106">TestResults</a> for the list of valid test results). There is also a `validationProps` object that you can stick arbitrary properties on to track state across draw loops.
+      This is a snippet of javascript that is run after every draw loop. Call
+      levelSuccess() in here to stop the level as a success. You can also
+      pass a number into levelSuccess() to give the student a non-perfect
+      test result (see <a href="https://github.com/code-dot-org/code-dot-org/blob/101fbfd2dcdd635d7359e991559ba782f9fd00be/apps/src/constants.js#L33-L106">TestResults</a>
```

Comment:
Oh, we should be able to use a HAML link here rather than an inline HTML one.  You could also hoist the URL out into a Ruby variable at the top of the file and use it here.

Reply:
Does this look reasonable now? I took the first of your suggestions but held off on the second to avoid a bunch more refactoring. How high priority is random tech debt like this, once it's found?

Reply:
I don't think we have an actively-applied, formal definition of priority for this kind of work.  If touching a file, it can be a good opportunity to tidy things up, though balanced against the risk of regression or of causing confusion by changing things unrelated to the core purpose of a PR/commit.  Other times, it can just be nice to tidy things up for the sake of it.